src/main/java/stirling/software/SPDF/utils/misc/InvertFullColorStrategy.java

```diff
@@ -98,8 +98,10 @@ private void invertImageColors(BufferedImage image) {
 
     // Helper method to convert BufferedImage to InputStream
     private File convertToBufferedImageTpFile(BufferedImage image) throws IOException {
-        File file = new File("image.png");
+        File file = File.createTempFile("image", ".png");
         ImageIO.write(image, "png", file);
+        //Ensures temp file is deleted when jvm terminates
+        file.deleteOnExit();
```

Comment:
Delete on exit isn't enough since this server could be running for weeks or months, we must delete it after operation
Can you replace this with a try finally?